test/fork/migration-v2.js

```diff
@@ -492,38 +543,89 @@ describe('v2 migration', function () {
     this.stakingPool = stakingPool;
   });
 
-  it.skip('block V1 staking', async function () {
+  it('block V1 staking', async function () {
     const tx = await this.pooledStaking.blockV1();
     await tx.wait();
   });
 
   it.skip('process all PooledStaking pending actions', async function () {
     let hasPendingActions = await this.pooledStaking.hasPendingActions();
+    let i = 0;
     while (hasPendingActions) {
+      console.log(`Calling processPendingActions. iteration ${i++}`);
       const tx = await this.pooledStaking.processPendingActions(100);
       await tx.wait();
       hasPendingActions = await this.pooledStaking.hasPendingActions();
     }
+    console.log('Done');
   });
 
-  it.skip('initialize TokenController', async function () {
-    const tx = await this.tokenController.initialize();
-    await tx.wait();
-  });
-
-  it.skip('unlock claim assessment stakes', async function () {
+  it('unlock claim assessment stakes', async function () {
     const stakesPath = path.join(__dirname, '../../scripts/v2-migration/output/eligibleForCLAUnlock.json');
     const claimAssessors = require(stakesPath).map(x => x.member);
+
+    const tcNxmBalance = await this.nxm.balanceOf(this.tokenController.address);
+
+    console.log('Token balances before running tc.withdrawClaimAssessmentTokens');
+    console.log({
+      tcNxmBalance: tcNxmBalance.toString(),
+    });
+
+    const totalToProcess = claimAssessors.length;
+    console.log(`Processing withdrawClaimAssessmentTokens for ${totalToProcess} claim assesors`);
+    let amountProcessed = 0;
+    while (claimAssessors.length > 0) {
+      const batchSize = 100;
+      const batch = claimAssessors.splice(0, batchSize);
+      await this.tokenController.withdrawClaimAssessmentTokens(batch);
+
+      amountProcessed += batchSize;
+      console.log(`Processed ${amountProcessed}/${totalToProcess}`);
+    }
+
     const tx = await this.tokenController.withdrawClaimAssessmentTokens(claimAssessors);
     await tx.wait();
   });
 
-  it.skip('transfer v1 assessment rewrds to assessors', async function () {
+  it('transfer v1 assessment rewards to assessors', async function () {
+    const tcNxmBalanceBefore = await this.nxm.balanceOf(this.tokenController.address);
+
     await this.claimsReward.transferRewards();
+
+    const tcNxmBalanceAfter = await this.nxm.balanceOf(this.tokenController.address);
+    const crNxmBalanceAfter = await this.nxm.balanceOf(this.claimsReward.address);
+
+    expect(crNxmBalanceAfter).to.be.equal(BigNumber.from(0));
+
+    const governanceRewardsMigrated = tcNxmBalanceAfter.sub(tcNxmBalanceBefore);
+
+    console.log({
+      governanceRewardsMigrated: governanceRewardsMigrated.toString(),
+    });
+    // expect(governanceRewardsMigrated).to.be.equal(this.governanceRewardsSum);
```

Comment:
There is a discrepancy here as well on the side of excess tokens. by a slight amount.

      -1106654884061072517264
      +870391213513961173071
      
      
      Extra tokens:
      
      236.26367054711136 NXM